scripts/installer/path.nsh

```diff
@@ -0,0 +1,190 @@
+# PATH utilities. Taken from:
```

Comment:
Should we change this dir to `win-installer` or even just `windows` instead?

Reply:
Renamed to `win-installer`.